main.cpp

```diff
@@ -35,88 +35,110 @@ arm.rotateCounterClockwise(); //demonstrates CamelCase
 
 /*
  1)
- Noun:
- action 1:
- action 2:
- action 3:
+ Noun: note
+ action 1: the note shifts up an octave
+ action 2: the note shifts down an octave
+ action 3: the note keeps a length of half note
  */
-
+note.shiftUpAnOctave();
+note.shiftDownAnOctave();
+note.keepALengthOfHalfNote();
 
 /*
  2)
- Noun:
- action 1:
- action 2:
- action 3:
+ Noun:quarter
+ action 1: the quarter starts
+ action 2: the quarter ends
+ action 3: the quarter suspends
  */
-
+quarter.start();
+quarter.end();
+quarter.suspend();
 /*
  3)
- Noun:
- action 1:
- action 2:
- action 3:
+ Noun:pro tools
+ action 1: the pro tools launches
+ action 2: the pro tools quits
+ action 3: the pro tools insert a new track
  */
-
+protools.launch();
+protools.quit();
+protools.insertANewTrack;
 
 /*
  4)
- Noun:
- action 1:
- action 2:
- action 3:
+ Noun:timer
+ action 1:the timer rings
+ action 2:the timer sets 
+ action 3:the timer resets
  */
-
+timer.ring();
+timer.set();
+timer.reset();
 /*
  5)
- Noun:
- action 1:
- action 2:
- action 3:
+ Noun:coffee maker
+ action 1: the coffee maker heats up
+ action 2: the coffee maker cleans up
+ action 3: the coffe maker brews coffee
  */
-
+coffeemaker.heatUp();
+coffeemaker.cleanUp();
+coffeemaker.brewCoffee();
```

Comment:
your noun should use camelCase for the naming
- [ ] apply camel case to noun name.